Makefile

```diff
@@ -58,6 +58,21 @@ image-multiarch:
 	docker buildx build --platform=$(PLATFORMS) -t $(REPO)/preupgradechecks:$(TAG) --push -f cmd/preupgradechecks/Dockerfile.fission-preupgradechecks .
 	docker buildx build --platform=$(PLATFORMS) -t $(REPO)/reporter:$(TAG) --push -f cmd/reporter/Dockerfile.reporter .
 
+multiarch-bundle:
+	docker buildx build --platform=$(PLATFORMS) -t $(REPO)/fission-bundle:$(TAG) --push -f cmd/fission-bundle/Dockerfile.fission-bundle .
```

Comment:
Hey @RealHarshThakur This seems a bit odd to me. So in platforms we have 3 but we are using same tag for all 3 arch images? I think what this will do is override the previous two and only last one will be available?

Reply:
Never mind - I think as pointed out in our Slac conversation - this will push all three arch images!